BUILDING.md

```diff
@@ -0,0 +1,128 @@
+```
+
+Full code:
+```ps1
+<#
+Note: This script hasn't been tested yet
+DISClAIMER: I don't know how to write powershell code :c
+#>
+
+function menu {
+    do {
+        do {
+            write-host ""
+            write-host "1. Bypass [Suggested] (To bypass the policy to get things installed)"
+            write-host "2. AllSigned (A bit more security)"
+            write-host ""
+            write-host "X - Exit"
+            write-host ""
+            write-host -nonewline "Type your choice and press Enter: "
+
+            $choice = read-host
+
+            write-host ""
+
+            $ok = $choice -match '^[12]+$'
+
+            if ( -not $ok) { write-host "Invalid selection" }
+        } until ( $ok )
+
+        switch -Regex ( $choice ) {
+            "1"
+            {
+                write-host("[DEBUG] Setting ExecutionPolicy to Bypass")
+                Set-ExecutionPolicy Bypass -Scope Process
+            }
+
+            "2"
+            {
+                write-host("[DEBUG] Setting ExecutionPolicy to AllSigned")
+                Set-ExecutionPolicy AllSigned
+            }
+        }
+    } until ( $choice -match "X" )
+}
+write-host("[DEBUG] Creating execpolicy variable")
+$execpolicy = Get-ExecutionPolicy 2>&1
+write-host("[DEBUG] execpolicy content: " + $execpolicy)
+
+write-host("[DEBUG] Checking if execpolicy is Restricted")
+if($execpolicy -eq "Restricted"){
+   write-host("[DEBUG] execpolicy is Restricted")
+   write-host("[DEBUG] Opening selection screen")
+   menu
+   write-host("[DEBUG] Closing selection screen")
+   write-host("[INFO] Just to be sure, run this again to check if it changed (:")
+
+} else {
+   write-host("[DEBUG] execpolicy is NOT Restricted")
+   write-host("[INFO] Ok, you're good now go on to the next step!")
+}
+```
+
+3. Install with powershell.exe
+
+
+Now execute the following script:
+
+```ps1
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+```
+
+3. After installing chocolatey, execute this script below to install the packages required.
+
+```ps1
+choco install nodejs git;
+```
+
+## macOS instructions
+
+_macOS Mojave (10.14) or later is required_
+
+### Installing prerequisites via Homebrew
+
+1. Install [homebrew](https://brew.sh/) by pasting the script below in a terminal. _(Skip this step if you already have homebrew)_
+
+```sh
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+2. After installing homebrew, execute this script below in a terminal to install the packages.
+
+```sh
+brew install node git
+```
+
+## Linux instructions
+
+### Installing prerequisites via built-in package managers
```

Comment:
This can stay as these are the preferred methods to install these applications and are by default included in the distro.